guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -42,21 +42,31 @@ For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy
 
 ifdef::katello,satellite[]
 +
-. Regenerate certificates.
-On the main {Project} server:
+. Regenerate certificates on your {ProjectServer}:
+.. Regenerate certificates for {SmartProxies} that use default certificates:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {certs-generate} --foreman-proxy-fqdn "myproxy.example.com" \
-                       --certs-update-all \
-                       --certs-tar "~/myproxy.example.com-certs.tar"
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+    --certs-update-all \
```

Comment:
```suggestion
--certs-update-all \
```

IMHO most of the time, we have no leading whitespace.

Reply:
Applied :+1: